zips/zip-0226.rst

```diff
@@ -463,16 +463,6 @@ $\mathsf{assetBurn}$ set is empty), the ''orchard_zsa_burn_digest'' is::
 
     BLAKE2b-256("ZTxIdOrcBurnHash", [])
 
-T.4b.i: assetBase
-.................
-The Asset Base being burnt encoded as the 32-byte representation of a point on the 
-Pallas curve.
-
-T.4b.ii: valueBurn
-..................
-Value of the Asset Base being burnt encoded as little-endian 8-byte representation 
-of 64-bit unsigned integer (e.g. u64 in Rust) raw value.
-
```

Comment:
Why remove completely?

Reply:
In ZIP 244 as well, when things are "field encoding bytes", the description is not given, because we know it is the encoding of the field